calcom/cal.com
# chore: redirect platform users to dashboard

## What does this PR do?



- This PR adds a redirect for all platform clients to make sure that a platform user is not able to access any of the consumer related pages.



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?
Run the webapp on localhost, go to `settings/platform/new` and make a new platform client. Once that is done and billing is setup for platform, if we try to navigate to any of the pages such as /event-types or /availability we should automatically be redirected to /settings/platform
